docs/code-reviews/process-guidance/author-guidance.md

```diff
@@ -22,6 +22,24 @@ Discuss design/code logic and address all comments as follows:
 - If you don't understand a comment, ask questions in the review itself as opposed to a private chat
 - If a thread gets bloated without a conclusion, have a meeting with the reviewer (call them or knock on door)
 
+## Use checklists
+
+At the moment of creating a pull request it is a good idea to put in the description of it a checklist of expected objectives to achieve in the PR. This way reviewers can be able to focus on specific areas and also consider some other tasks as secondary according to this list.
+
+### Get a process for fixing found issues
+
+Before getting started with any commit, your team should define how you will measure the effectiveness of peer review and get some goals. Using a [SMART criteria](https://en.wikipedia.org/wiki/SMART_criteria) the team can get this objectives in the most accurate way and not something very general like **fix more bugs**, the more precise a goal can be the easier it will be to hit the target.
+
+Metrics can play a significant role in this part, you can include an **Inspection rate** (the speed with which review is performed), **Defect rate** (the number of bugs found within an hour of review) and **Defect density** (the number of bugs found per piece of code) as the most common ones to be considered.
+
+## Code should have annotations before the review
+
+If you can't avoid large PRs, include explanations of the changes in order to make it easier for the reviewer to review the code, with clear comments the reviewer can identify the goal of every code block.
+
 ## Track progress
 
 - If the reviewers have not responded in a reasonable time (generally a day or two), ping them or raise the issue in a daily meeting.
+
+## Foster a positive code review culture
```

Comment:
This is more a reviewer guidance than an author guidance - was it put here by mistake?

Reply:
Well, goal here was to let the author know it's ok to ping the reviewer and let him know there's some code there to be reviewed and probably unblock this in a faster way.

Reply:
Are we referring to the same section here? This was in regards to 

"Foster a positive code review culture

Code reviews play a critical role in product quality and it should not represent an arena for long discussions or even worse a battle of egos. What matters is a bug caught not who made it, not who found it, not who solved it. The only thing that matters is having the best possible product."

Reply:
Well, not that you say it that way yes, it seems to fit.